src/automation/azext_automation/aaz/latest/automation/python3package/_create.py

```diff
@@ -0,0 +1,305 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "automation python3package create",

```

Comment:
Please separate different words with `-`
```suggestion
    "automation python3-package create",
```

Reply:
In addition, if we want to expand multiple different package types in the future, could we design this command as `automation package create` and `python3` as one of the valid values of the `--package-type` parameter? 
This design may have better scalability

Reply:
I have changed name as suggested